docs/roadmap.md

```diff
@@ -53,80 +42,59 @@ It focuses on enabling three main use cases:
+| Supports Unpackaged (non-MSIX) Deployment              |                    |                    | &#128994;                       |
+| Works downlevel to Windows 10 version 1809 and above   | &#128994;          | &#128994;          | &#128994;                       |
+| Supports the latest .NET                               | &#128994;          | &#128994;          | &#128994;                       |
+| ARM64 support                                          | &#128994;          | &#128994;          | &#128994;                       |
+| `<SwapChainPanel> `                                    | &#128994;          | &#128994;          | &#128994;                       |
+| Intellisense, Hot Reload, Live Visual Tree             | &#128311;          | &#128311;          | &#128994;[^1]                   |
+| Chromium-based WebView2                                | &#128994;          | &#128994;          | &#128994;                       |
+| Title bar customization                                | &#128994;          | &#128994;          | &#128994;                       |
+| Fluent Shadows                                         | &#128994;          | &#128994;          | &#128994;                       |
+| Input validation for data fields                       | &#128311;          | &#128311;          | &#128311;                       |
+| Supports multiple top-level windows on the same thread | &#128311;          | &#128311;          | &#128311;                       |
+| Support multiple top-level windows on separate threads | &#128311;          | &#128311;          | &#128311;                       |
+| Drag and drop                                          | &#128994;          | &#128994;          | &#128994;                       |
+| RenderTargetBitmap                                     | &#128994;          | &#128994;          | &#128994;                       |
+| Mouse cursor customization                             | &#128994;          | &#128994;          | &#128994;                       |
+| Animated Gif support                                   | &#128994;          | &#128994;          | &#128994;                       |
+| VirtualSurfaceImageSource (VSIS) support               | &#128994;          | &#128994;          | &#128994;                       |
+| In-app acrylic                                         | &#128994;          | &#128994;          | &#128994;                       |
+| XAML Islands                                           |                    | &#128311;          | &#128311;                       |
 
-### Legend
-&#128994; - Available as supported feature
+[^1]: Supported in packaged apps only
 
-&#128311; - Available as experimental feature 
+## WinUI 2 Controls Library
 
-&#129002; -  Stretch goal 
+WinUI 2 is native user experience (UX) framework for both Windows desktop and UWP applications tightly integrated with Windows 10 and above SDKs. 
```

Comment:
Should it be "WinUI 2 is _a_ native user experience framework"?

Reply:
Yes, good catch, fixed :)